dashboard/test/models/pd/international_opt_in_test.rb

```diff
@@ -29,7 +22,7 @@ class Pd::InternationalOptInTest < ActiveSupport::TestCase
     refute build(:pd_international_opt_in, form_data: {}.to_json, user_id: teacher.id).valid?
 
     refute build(
-      :pd_international_opt_in, form_data: FORM_DATA.merge({ages: nil}).to_json, user_id: teacher.id
+      :pd_international_opt_in, form_data: FORM_DATA.merge({schoolName: nil}).to_json, user_id: teacher.id
```

Comment:
This is a change I don't quite understand. What changed exactly to assert something so different in this test? I'm sure its just smoothing out the content changes and the requested data, but it is a surprising change when not much changed around it in the test.

Reply:
My understanding was that this test evaluates an invalid required input, to make sure it does not allow the form submission without that response. I arbitrarily picked a different required field after removing the ages field.

Reply:
That was my intuition and that makes sense to me!